src/galax/units.py

```diff
@@ -56,8 +72,17 @@ class UnitSystem:
         u.get_physical_type("angle"),
     ]
 
-    # TODO: type hint `units`
-    def __init__(self, units: Any, *args: u.UnitBase) -> None:
+    def __init__(
+        self,
+        units: Union[
+            u.UnitBase,
+            u.Quantity,
+            dict[str, u.UnitBase],
+            dict[str, u.Quantity],
```

Comment:
I think it can also be a mix.

```suggestion
            dict[str, u.UnitBase | u.Quantity],
```

Except, looking to line 70, it doesn't look like `dict` works at all.
At a glance, https://github.com/adrn/gala/blob/main/gala/units.py doesn't accept dicts either.

Reply:
Whoops - fixed in next commit!